example/scripts/tests-parallel.sh

```diff
@@ -14,7 +14,7 @@ testOutputDir=scripts/test_output
 rm -r -f $testOutputDir
 mkdir -p $testOutputDir
 
-echo "" | ./node_modules/.bin/parallel -j 4 sh scripts/run_test.sh  $testOutputDir ::: $TESTS
```

Comment:
From talking to @laurenfrederick, this was an accidentally-committed change. Switching it back to "0".